### ISSUE TITLE
Blocking PMs is broken 

### ISSUE THREAD

**Issue by halamix2** [PHP-Outburst/myUPB#39]
pmblocklist.php?action=add&user_id=**x** is empty, only footer gets loaded but it blocks user successfully.
However unblocking doesn't work, clicking on Unblock button on pmblocklist.php does nothing, doesn't even reload the page

(forum post: http://www.basildon.com/upb/viewtopic.php?id=6&t_id=7 )

**Comment by halamix2** [comment]
Also:clicking on PM from blocked user (viewpm.php?section=inbox&id=**x**) leads to
`Fatal error: Uncaught Error: 
Call to undefined function getUsersPMBlockedList() in /viewpm.php:75
Stack trace: #0 {main} thrown in /viewpm.php on line 75`

On sender side:
`Fatal error: Uncaught Error: Call to undefined function user_date() in /home/halamix2/domains/halamix2.pl/public_html/myupb/viewpm.php:108 Stack trace: #0 {main} thrown in /home/halamix2/domains/halamix2.pl/public_html/myupb/viewpm.php on line 108`

**Comment by phpcodex** [comment]
I'm experiencing another issue before I get here, I've started the journey from the beginning and sent my first PM etc.. 

A) so now I'm blocking my first message-sender, and here I have a near blank page, which needs fixing, although it does 'block' the user (or appears to) and 

B) when I attempt to unblock the user, the post doesn't work. Nothing is getting submitted.

so looking at issue A first.

**Comment by phpcodex** [comment]
https://github.com/PHP-Outburst/myUPB/pull/45
